spec/index.bs

```diff
@@ -704,18 +704,18 @@ This specification defines the following [=usage scopes=]:
   3. the whole {{GPURenderPassEncoder}}.
 
 <div class=note>
-  Resources that are used in state setting calls are always added to current [=usage scope=] as used resources.
-  This means the following example usages should be validated in current [=usage scope=]:
-
-  - Resources used in every {{GPUProgrammablePassEncoder/setBindGroup(index, bindGroup, dynamicOffsets)|GPUProgrammablePassEncoder.setBindGroup}}
-    regardless of whether the shader or {{GPUPipelineLayout}} actually depends on these bindings, or the bind group is shadowed by another 'set' call.
-  - Vertex buffer used in every {{GPURenderEncoderBase/setVertexBuffer()|GPURenderEncoderBase.setVertexBuffer}}
-    regardless of whether draw call depends on this buffer, or this buffer is shadowed by another 'set' call.
-  - Index buffer used in every {{GPURenderEncoderBase/setIndexBuffer()|GPURenderEncoderBase.setIndexBuffer}}
-    regardless of whether draw call depends on this buffer, or this buffer is shadowed by another 'set' call.
-  - Textures used as color attachments, resolve attachments or depth/stencil attachment in {{GPURenderPassDescriptor}} by {{GPUCommandEncoder/beginRenderPass()}}
-    regardless of whether the shader actually depends on these attachments.
-  - Resources used in bind group entries with visibility 0, or visible only to the the compute stage but used in a render pass or vice versa.
+    Resources that are used in state setting calls are always added as used resources in the containing [=usage scope=] as used resources.
+    This means the following example resource usages **are** included in usage scope validation:
+
+    - Resources used in every {{GPUProgrammablePassEncoder/setBindGroup(index, bindGroup, dynamicOffsets)|GPUProgrammablePassEncoder.setBindGroup}}
+        regardless of whether the shader or {{GPUPipelineLayout}} actually depends on these bindings, or the bind group is shadowed by another 'set' call.
+    - Vertex buffer used in every {{GPURenderEncoderBase/setVertexBuffer()|GPURenderEncoderBase.setVertexBuffer}}
+        regardless of whether draw call depends on this buffer, or this buffer is shadowed by another 'set' call.
+    - Index buffer used in every {{GPURenderEncoderBase/setIndexBuffer()|GPURenderEncoderBase.setIndexBuffer}}
+        regardless of whether draw call depends on this buffer, or this buffer is shadowed by another 'set' call.
+    - Textures used as color attachments, resolve attachments or depth/stencil attachment in {{GPURenderPassDescriptor}} by {{GPUCommandEncoder/beginRenderPass()|GPUCommandEncoder.beginRenderPass}}
```

Comment:
Best to match the default formatting which shows the `()`:

```suggestion
    - Textures used as color attachments, resolve attachments or depth/stencil attachment in {{GPURenderPassDescriptor}} by {{GPUCommandEncoder/beginRenderPass()|GPUCommandEncoder.beginRenderPass()}}
```

Reply:
Got it. The spec is not consistent on this style issue. So I don't know which one is the best. Some of them don't use `()` at all. Some of them use one `()`. And the rest of them use two `()`s.

Reply:
We shouldn't have ones with two `()`s. I only see one instance of that right now (just searching in the spec for `)(`) but are there others?

The `()` marker is best because it clearly identifies that it's a function. We usually xref functions using `functionname()` but sometimes Bikeshed requires us to write `functionname(arg, list)` if there are overloads. Those are fairly rare though.

Reply:
Sorry, I mean one before the `|`, and one after the `|` when there are two `()`s.  
Currently, the styles in spec to refer to a function could be:
1) {{GPUBuffer/getMappedRange}}  // no `()`
2) {{GPUBuffer/destory()}} // one `()`
3) {{GPUCommandEncoder/beginRenderPass()|GPUCommandEncoder.beginRenderPass}} // one `()`
4) {{GPUCommandEncoder/beginRenderPass()|GPUCommandEncoder.beginRenderPass()}} // two `()`s.  

But I don't know which one the the best. 

If we have `()`, some of them may list parameters, and the others don't. I suppose this differentiation is used for overloads.

Reply:
Ah, I see. I think we should use 2 or 4 depending on the context. Probably best to use 4 in most places; I just wish it didn't have to be so verbose.